UI/window-basic-main-outputs.cpp

```diff
@@ -556,7 +556,7 @@ void SimpleOutput::UpdateStreamingSettings_amd(obs_data_t *settings,
 	obs_data_set_int(settings, "Usage", 0);
 	obs_data_set_int(settings, "Profile", 100); // High
 	obs_data_set_string(settings, "profile", "high"); // High
-	
+
```

Comment:
This is likely unintentional but probably desired due to the .editorconfig in use for OBS. Not all files are conforming to the .editorconfig format yet, but when saving a modified file, it will enforce some of the rules (in this case, no trailing whitespace) throughout the entire file.